src/ComfortPay/Message/CheckCardResponse.php

```diff
@@ -2,13 +2,18 @@
 
 namespace Omnipay\ComfortPay\Message;
 
+use Omnipay\ComfortPay\Gateway;
 use Omnipay\Common\Message\AbstractResponse;
 use Omnipay\Common\Message\ResponseInterface;
 
 class CheckCardResponse extends AbstractResponse implements ResponseInterface
 {
     public function isSuccessful()
     {
-        return $this->data == 0;
+        return in_array($this->data->status, [
```

Comment:
I think, that (looking at the original implementation), this should be *successful* only if the resulting status is `CARD_STATUS_OK`. The old implementation returned `0` only if the card was still valid.

Reply:
You can probably extend the response with additional `getStatus()` method in case client would like to see specific status, but primarily you probably don't want to make this breaking change.